civihr_employee_portal/templates/civihr-employee-portal-civihr-report-custom.tpl.php

```diff
@@ -27,110 +27,120 @@ class="panel-body-wrap panel-body-wrap--collapse"
   </div>
```

Comment:
Seems you have an extra `<div>` before you close the `if`, which seems to be closing the root element, please check

```html
<div id="civihrReports">
  <?php if (!empty($filters)): ?>
    <div ng-controller="FiltersController as filters">
    </div>
  </div>
<?php endif; ?>
```